src/components/ha-sidebar.html

```diff
@@ -197,6 +201,10 @@
     this._boundUpdateStyles = this.updateStyles.bind(this);
   },
 
+  tr: function (pg, text, options) {
```

Comment:
Okay I spend way too long this evening researching all this stuff. Even went on the Polymer slack to ask for advice (but learned nothing new).

You are right that 15kb is a lot. It's 5kb gzipped, but still, a lot. (The app core is 2.8kb gzip)

However, since internationalization is a big can of worms, it is probably worth it. Because we'll have to localize settings, every card, more info and current + future panels…